libs/web/modules/src/dashboard/user/registrasi/biodata/section/data-nilai/index.tsx

```diff
@@ -87,8 +87,8 @@ export const DataNilaiSection: FC = (): ReactElement => {
     }
   });
 
-  const labelSemesterStyle = "text-[10px] md:text-base";
-  const labelCourseStyle = "text-[8px] lg:text-base font-bold pr-4 lg:pr-0";
+  const labelSemesterStyle = "text-[13px] md:text-base";
```

Comment:
jadikeun tailwind alias di tailwind config

https://stackoverflow.com/questions/73898178/aliases-for-tailwind-classes